packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -208,16 +209,16 @@ class SemanticsData with Diagnosticable {
     this.customSemanticsActionIds,
   }) : assert(flags != null),
        assert(actions != null),
-       assert(label != null),
-       assert(value != null),
-       assert(decreasedValue != null),
-       assert(increasedValue != null),
-       assert(hint != null),
-       assert(label == '' || textDirection != null, 'A SemanticsData object with label "$label" had a null textDirection.'),
-       assert(value == '' || textDirection != null, 'A SemanticsData object with value "$value" had a null textDirection.'),
-       assert(hint == '' || textDirection != null, 'A SemanticsData object with hint "$hint" had a null textDirection.'),
-       assert(decreasedValue == '' || textDirection != null, 'A SemanticsData object with decreasedValue "$decreasedValue" had a null textDirection.'),
-       assert(increasedValue == '' || textDirection != null, 'A SemanticsData object with increasedValue "$increasedValue" had a null textDirection.'),
+       assert(attributedLabel != null),
+       assert(attributedValue != null),
+       assert(attributedDecreasedValue != null),
+       assert(attributedIncreasedValue != null),
+       assert(attributedHint != null),
+       assert(attributedLabel.string == '' || textDirection != null, 'A SemanticsData object with label "${attributedLabel.string}" had a null textDirection.'),
```

Comment:
This looks kinda strange. Why would you provide an AttributedString with an empty `string` property?

Reply:
There is always a default value from semanticsconfiguration. after this point, the label is non null